environment.integration

```diff
@@ -14,7 +14,7 @@ API_DOMAIN_NAME="dss.${DCP_DOMAIN}"
 DSS_GCP_SERVICE_ACCOUNT_NAME="org-humancellatlas-integration"
 DSS_ZONE_NAME="integration.data.humancellatlas.org."
 ACM_CERTIFICATE_IDENTIFIER="390e82ea-a684-49f4-a4b3-857f22cee874"
-DSS_CHECKOUT_BUCKET_OBJECT_VIEWERS="serviceAccount:1037839730885-compute@developer.gserviceaccount.com,serviceAccount:caas-account@broad-dsde-mint-test.iam.gserviceaccount.com,serviceAccount:caas-prod-account-for-test@broad-dsde-mint-test.iam.gserviceaccount.com,serviceAccount:caas-prod-account-for-int@broad-dsde-mint-integration.iam.gserviceaccount.com,serviceAccount:cromwell-metadata-uploader@broad-dsde-mint-integration.iam.gserviceaccount.com"
+DSS_CHECKOUT_BUCKET_OBJECT_VIEWERS="serviceAccount:1037839730885-compute@developer.gserviceaccount.com,serviceAccount:caas-account@broad-dsde-mint-test.iam.gserviceaccount.com,serviceAccount:caas-prod-account-for-int@broad-dsde-mint-integration.iam.gserviceaccount.com,serviceAccount:cromwell-metadata-uploader@broad-dsde-mint-integration.iam.gserviceaccount.com,serviceAccount:bluebox-subscription-manager@broad-dsde-mint-integration.iam.gserviceaccount.com"
```

Comment:
@rhiananthony  i did not see any reference of the blubox-subscription manager in the DSS codebase/config files, did you need it added to the checkout bucket object viewers?

Reply:
https://github.com/HumanCellAtlas/data-store/issues/2284#issuecomment-521724783